lib/config/default.config.js

```diff
@@ -250,7 +250,7 @@ module.exports = {
       }
     },
     internalIndex: {
-      bootstrapLockTimeout: 5000
+      bootstrapLockTimeout: 60000
```

Comment:
Why this change?

Reply:
I don't know this is coming from @scottinet's commit

Reply:
Yes it does.
I cherry-picked a commit from the 2-dev branch to master to fix our CI a few days ago. I don't get why it appears here. :thinking: 
But it doesn't really matter.